docs/edit-tool.md

```diff
@@ -154,6 +155,18 @@ In Addtional to the default settings for a tool, you can specify the following o
 
 ![Route Planner Panel](route-planner.png)
 
+## Geomark
+
+The Geomark tool allows creating and loading a geomark, which is an area of interest that is stored in a service and can be shared by a URL. A geomark can be created in SMK by drawing one or more shapes and saving the drawing as a geomark. A geomark can also be loaded by its URL.
```

Comment:
Consider "The Geomark tool allows users to create and load geomarks. A geomark is an area of interest that is stored in a web service and can be shared by a URL.

Reply:
I've broken up the longer sentence into two and specified "web service".